lilypad/server/services/base.py

```diff
@@ -24,6 +25,7 @@ def find_record_by_id(self, id: int | str) -> _TableT:
         """Find record by id"""
         record_table = self.session.exec(
             select(self.table).where(
+                self.table.organization_uuid == self.user.active_organization_uuid,  # pyright: ignore[reportAttributeAccessIssue]
```

Comment:
is there an issue here because `self.user` is not defined? can we define it so there isn't a type error?

I see it in the init, why not include it as an attribute of the class?

Reply:
type error is on self.table where organization_uuid is not defined. every table would need to extend from a base model with organization_uuid

Reply:
should we not do that? seems like here we're expecting every model to have it? this will crash if not, right?

Reply:
online discussion: leaving as-is and fixing in the next PR